README.md

```diff
@@ -1 +1,32 @@
-# mongodb-operator
\ No newline at end of file
+# Charmed MongoDB Operator
+
+
+## Description
+
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database.
+
+MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB on physical or virtual machines.
+
+
+## Usage
+
+As this operator charm is not yet published, you need to follow the build & deploy instructions from [CONTRIBUTING.md](./CONTRIBUTING.md).
+
+
+## Relations
+
+Supported [relations](https://juju.is/docs/olm/relations):
+
+- This charm supports peer relations, wich provides replica sets for the database.
+
+The charm will implement both [sharding](https://docs.mongodb.com/manual/sharding/) and [replica set](https://docs.mongodb.com/manual/replication/). To do this, it will need an extra component, the `config-server`.
+
+Future relations will include:
+
+- Mongo router `mongos` charm deployed alongside MongoDB charm. It will handle a sharded deployment, and it can work as a MongoDB database interface.
+- A MongoDB database `shard` interface. It will be used by the mongos charm as well.
+- A `config-server` interface used by the mongos charm.
+
+## Contributing
+
+TODO: update when charm is published
```

Comment:
I could add the same from the template here: 
```suggestion
Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhancements to this charm following best practice guidelines, and [CONTRIBUTING.md](./CONTRIBUTING.md) for developer guidance.
```

Reply:
@zmraul I think it would be more helpful to link the the CONTRIBUTING.md since that has all the information they need to know about contribution (and it will have the Juju SDK docs and contributing process)